Sources/Components/Panel/Panel.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public class Panel: UIView {
+    private let cornerRadius: CGFloat = 8
+
+    private lazy var textLabel: Label = {
+        let label = Label(style: .body, withAutoLayout: true)
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.textAlignment = .center
+        label.textColor = style.textColor
+        return label
+    }()
+
+    private let style: Panel.Style
+
+    public init(style: Panel.Style) {
+        self.style = style
+        super.init(frame: CGRect(x: 0, y: 0, width: 320, height: 20))
```

Comment:
Do we need to set the frame for this one?

Reply:
Probably not 👍